Added a lot of missing spring items and some winter2022 items.

Boontato opened the pull request:
<!--
To make reviewing easier, please fill out the table with the IDs.
Detailing changes on the Notes column is appreciated:
E.g:
  Season 2
  Specials (2-5)
  Movie
  TVDB Removed \ Reordered Episodes

TVDB URLs are prefered in the example format (with ID) instead of their address bar redirect (title slug):
   👎 https://thetvdb.com/series/those-obnoxious-aliens/
   👍 https://thetvdb.com/index.php?tab=series&id=75113
-->

| AniDB | TVDB/TMDB/IMDB | Notes |
| --- | --- | --- |
| https://anidb.net/anime/16320 | https://thetvdb.com/index.php?tab=series&id=402893 | Season 1 |
| https://anidb.net/anime/16943 | https://thetvdb.com/index.php?tab=series&id=415509 | Season 1 |
| https://anidb.net/anime/17147 | https://thetvdb.com/index.php?tab=series&id=414965 | Season 1 |
| https://anidb.net/anime/17301 | https://thetvdb.com/index.php?tab=series&id=424278 | Season 1 |
| https://anidb.net/anime/17351 | https://thetvdb.com/index.php?tab=series&id=430212 | Season 1 |
| https://anidb.net/anime/17633 | https://thetvdb.com/index.php?tab=series&id=397280 | Season 2 |
| https://anidb.net/anime/17711 | https://thetvdb.com/index.php?tab=series&id=427027 | Season 1 |
| https://anidb.net/anime/17739 | https://thetvdb.com/index.php?tab=series&id=427708 | Season 1 |
| https://anidb.net/anime/17800 | https://thetvdb.com/index.php?tab=series&id=432849 | Season 1 |
| https://anidb.net/anime/17867 | https://thetvdb.com/index.php?tab=series&id=431237 | Season 1 |
| https://anidb.net/anime/17971 | https://thetvdb.com/index.php?tab=series&id=428539 | Season 1 |


<!-- EXAMPLE ROWS - Enjoy CopyPaste
| https://anidb.net/anime/ | https://thetvdb.com/index.php?tab=series&id= | Season X |
| https://anidb.net/anime/ | https://www.imdb.com/title/tt <br/> https://www.themoviedb.org/movie/ | Movie |
EXAMPLES END -->

BrutuZ commented:
Uma Musume has a dupe (#344220 Season 3), I wonder which will survive TVDB's mods' mood swings.

Last one is a movie (Gekijouban), both TVDB seasons have multiple episodes

Boontato commented:
ah gotcha, I updated ume musume to follow season 3 since at least that has thumbnails so far,
also updated pole princess, didn't know a "movie" existed but meant to map the show. movie doesn't come out till december 2023
